### PR TITLE
refactor index.js a bit for readability

### PR DESCRIPTION
**Pull request by staltz**
- Renamed some constants to use underscores
- Used `const` wherever possible
- Moved the exports to the bottom of the file

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -132,7 +124,7 @@ exports.encode = function encode(feedformat, value) {
       return encoder.box(value)
     else
       return encoder.string(value)
-  } else if (typeof value == "boolean") {
+  } else if (typeof value === "boolean") {
```
Comment (staltz) [inline index.js:127]: We should never have double equality

File: index.js [hunk 2]
```diff
@@ -1,34 +1,34 @@
...
+const FEED_TYPE = Buffer.from([0])
+const CLASSIC_FEED_TYPE = Buffer.from([0, 0])
+const GG_FEED_TYPE = Buffer.from([0, 1])
+const BB_FEED_TYPE = Buffer.from([0, 3])
+
+const MSG_TYPE = Buffer.from([1])
+const CLASSIC_MSG_TYPE = Buffer.from([1, 0])
+const GG_MSG_TYPE = Buffer.from([1, 1])
+const BB_MSG_TYPE = Buffer.from([1, 4])
 const nullMsgBytes = Buffer.alloc(32)
```
Comment (mycognosist) [inline index.js:15]: Any reason for leaving this as camel case? Might be more consistent as `NULL_MSG_BYTES`?

**Review by mycognosist — APPROVED** [review]
LGTM. Underscores in the constant names help readability a lot. I have nearly-identical names in `ssb-bfe-rs` (will probably change my `BOX_1_TYPE` -> `BOX1_TYPE` to match the naming here).

Also: +1 for having all exports together at the bottom.

File: index.js (hunk 2 above)
Comment (staltz) [reply index.js:15]: Yeah, I thought about that for a while too, I'm not sure it belongs to the same category as the others, and I didn't author this code so I felt maybe we should keep it like this. We can change it, but perhaps in another PR.

**Pull request merged**